R/dataframe-s3-base.R

```diff
@@ -151,7 +160,7 @@ tail.polars_data_frame <- function(x, n = 6L, ...) x$tail(n = n)
     if (!missing(drop)) {
       warn(
         c(
-          `!` = "`drop` argument ignored for subsetting a frame with `x[j]`.",
+          `!` = "`drop` argument ignored for subsetting a DataFrame with `x[j]`.",
```

Comment:
This is never hit when `x` is a LazyFrame because in that case `drop` is missing when we dispatched to `[.polars_data_frame`.